### PR TITLE
Minimise config

### PR DESCRIPTION
**Pull request by jutaro**
Minimise the default tracer file config, by adding a hardcoded default tracer config. The config loaded from the file can then overwrite all values set by defaults. This pull requests then includes some adoptions to config files. As well some fixes for metrics are included in this pull request.

### CODE REVIEW

File: doc/New Tracing Quickstart.md [hunk 1]
```diff
@@ -20,19 +20,21 @@ on the namespace, and require a one-to-one correspondence between namespaces and
 
 As we have two mechanisms for the same purpose for historic reasons, we will soon
 __deprecate the Kind field__, and it will disappear in the near future. So we strongly
-advice to use namespaces for any analysis tools of traces!  
+advice to use namespaces for any analysis tools of traces!
```
Comment (newhoggy) [inline doc/New Tracing Quickstart.md:23]: `s/advice/advise`

File: trace-dispatcher/doc/trace-dispatcher.md [hunk 2]
```diff
@@ -555,6 +555,8 @@ message per second, then add this to your configuration file:
     maxFrequency: 1.0
 ```
 
+In Cardano a default configuration is given in the module Cardano.Node.Tracing.DefaultTraceConfig. In the config file all entries of the default configuration can be overridden. To remove a frequency limiter, define a limiter with maxFrequency 0.0.
```
Comment (newhoggy) [inline trace-dispatcher/doc/trace-dispatcher.md:558]: `Cardano.Node.Tracing.DefaultTraceConfig` is worth backticking.

**Review by newhoggy — APPROVED** [review]

File: doc/New Tracing Quickstart.md (hunk 1 above)
Comment (jutaro) [reply doc/New Tracing Quickstart.md:23]: Okay, learned some more English

File: trace-dispatcher/doc/trace-dispatcher.md (hunk 2 above)
Comment (jutaro) [reply trace-dispatcher/doc/trace-dispatcher.md:558]: Okay

**Review by deepfire — APPROVED** [review]
Looks good, thank you @jutaro !

Now we need to benchmark this.

**Pull request merged**